README.md

```diff
@@ -121,6 +124,15 @@ See the [Tutorials](https://vizro.readthedocs.io/en/stable/pages/tutorials/first
 
 See the [Documentation](https://vizro.readthedocs.io/en/stable/) for more details
 
+## Vizro-AI: The Innovative Extension of Vizro
+
+<img src="./vizro-ai/docs/assets/readme/readme_animation.gif" alt="Gif to demonstrate vizro-ai">
+
+Vizro-AI is a tool designed for generating data visualizations.
+It serves as an extension to Vizro, leveraging natural language capabilities to empower users in creating charts effortlessly.
+
+See [Vizro-AI Documentation](https://vizro-ai.readthedocs.io/en/latest/) for more details.
+
```

Comment:
I think I would remove it from here as it looks a bit misplaced

```suggestion
```

Reply:
Discussed this with Joe and we probably would like to retain the Gifs and introduction here to draw some attractions as this is the main pages? 

as what after documentation should be common for vizro and vizro-ai